utils_nlp/models/transformers/common.py

```diff
@@ -1,4 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
 
+from pytorch_transformers.modeling_bert import BERT_PRETRAINED_MODEL_ARCHIVE_MAP
+from pytorch_transformers.modeling_xlnet import XLNET_PRETRAINED_MODEL_ARCHIVE_MAP
+from pytorch_transformers.modeling_roberta import ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP
+
+BERT_PRETRAINED_MODEL_ALL = list(BERT_PRETRAINED_MODEL_ARCHIVE_MAP.keys())
```

Comment:
Looks good just remove list(..). Dictionary `keys()` already return a list

Reply:
Dictionary keys() returns dict_keys type. It's iterable, but a regular list is probably better.

Reply:
you can remove the .keys() though :)

Reply:
Wow, why is that possible? I think I'm going to close this PR for now and do something like you did in the seq classifier PR.

Reply:
a list can take an iterable object, and converts it to a list by accessing the __iter__ function, which in the case of dictionaries, returns the keys